src/tests/Login.test.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import { render, fireEvent, screen, waitForElement } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import Login from "../login/Login";
+import { act } from 'react-dom/test-utils';
+
+
+const server = setupServer(
+    rest.post('http://127.0.0.1:5000/login', (req, res, ctx) => {
+        const payload = {
+            username: "MyUsername",
+            password: "12345678",
+        };
+        expect(req.body).toEqual(payload)
+        return res(ctx.json({ 
+            access_token: "fake_access_token",
+            access_expiry: 1594771200
+        }))
+    })
+)
+
+beforeAll(() => server.listen())
+afterEach(() => server.resetHandlers())
+afterAll(() => server.close())
+
+
+
+it('allows the user to login successfully', async () => {
```

Comment:
We might want to have another test that checks if on successful login, redirect to My Space happens. That test might be tricky, so we can have a separate PR for that, not on priority though, so as not to block you